docs/apertures.rst

```diff
@@ -161,6 +161,24 @@ of 0.5 and a normalizing flux of ``FLUX_AUTO``. The equivalent here is::
     xwin, ywin, flag = sep.winpos(data, objs['x'], objs['y'], sig)
 
 
+Segmentation-masked image measurements
+--------------------------------------
+
+SourceExtractor provides a mechanism for measuring the "AUTO" and "FLUX_RADIUS" parameters for a given object including a mask for neighboring sources.  In addition to the mask, setting the SourceExtractor parameter ``MASK_TYPE=CORRECT`` further fills the masked pixels of a given source with "good" pixel values reflected opposite of the masked pixels.  The ``SEP`` photometry and measurement functions provide an option for simple masking without reflection or subtracting neighbor flux.  
+
+For example, using a segmentation array provided by ``sep.extract``, we can compute the masked ``flux_radius`` that could otherwise be artificially large due to flux from nearby sources::
+
+    # list of object id numbers that correspond to the segments
+    objs['id'] = np.arange(len(objs), dtype=np.int32)
```

Comment:
In the segmentation array from `sep.extract`, the first object's pixels have value 1. So, I think this should be `np.arange(1, len(objs)+1, ...)`?

Reply:
Yes, that's right.